src/avro_json_encoder.erl

```diff
@@ -323,6 +364,24 @@ to_hex(D) when D >= 10 andalso D =< 15 ->
 encode_float(Number) ->
   ?INLINE(iolist_to_binary(io_lib:format("~p", [Number]))).
 
+%% TODO: this was taken from avro.erl, where it is private
```

Comment:
move it to avro_util

Reply:
then I realise there is no need for this function to be here neither need to move it. see comments above.

Reply:
Removed